client/src/components/Counter.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ReactComponent as HeartIcon } from "../assets/icons/heart-icon.svg";
+import styled from "@emotion/styled";
+
+export default function Counter(props) {
```

Comment:
You should think of naming your props for a better code understanding
```suggestion
export default function Counter(some name) {
```

If you like, you could also destructre you props here
```suggestion
export default function Counter({number}) {
```